src/ucp/rndv/proto_rndv.c

```diff
@@ -707,7 +708,7 @@ ucp_proto_rndv_send_reply(ucp_worker_h worker, ucp_request_t *req,
         rkey           = NULL;
     }
 
-    ucp_proto_select_param_init(&sel_param, op_id, op_attr_mask, 0,
+    ucp_proto_select_param_init(&sel_param, op_id, op_attr_mask, op_flags,
```

Comment:
Should we do the same fix in master branch?

Reply:
yes, together with relevant gtest